workspaces/quay/plugins/quay/.eslintrc.js

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright 2024 The Backstage Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+const eslintConfigFactory = require('@backstage/cli/config/eslint-factory');
+
+module.exports = eslintConfigFactory(__dirname); // Load the default Backstage config
+module.exports.rules = {
+  // TODO: Remove this later
+  'react/forbid-elements': 'off',
+};
```

Comment:
Community plugins uses this:

```suggestion
module.exports = require('../../.eslintrc.cjs');
```

The rules = {} would remove also all predefined rules. I wanted disable the `react/forbid-elements` in my Tekton PR as well and used this instead:

https://github.com/backstage/community-plugins/pull/1914/files#diff-f8f6becee853d2365355a2133dcb24e236ef855295525894ea8dd8cc0999d06d

Maybe you can pick up the same here?

Reply:
Added this to workspaces/quay/.eslintrc.js as you did for the tekton PR!